site/com.jboss.devstudio.core.p2.inf

```diff
@@ -13,13 +13,6 @@ units.1.requires.1.name=com.jboss.devstudio.core.package
 units.1.requires.1.range=[$version$,$version$]
 units.1.requires.1.greedy=true
 
-requires.0.namespace=org.eclipse.equinox.p2.iu
```

Comment:
Is it safe to remove it instead of update ?

Reply:
update fails on config.a.jre.javase update, but not on a.jre.javase. updating PR